action.yml

```diff
@@ -79,9 +79,6 @@ inputs:
   paths_no_filter:
     description: 'A list of file paths to be included regardless of the filter (newline/comma separated)'
     required: false
-  auto_approved_user:
```

Comment:
without digging through the codebase, i'm guessing the `login` param is used elsewhere and doesn't need to be removed?

Reply:
Yep exactly! Also going to wait until I coordinate with someone who is the only known user of this helper since it's a breaking change and I'm cheating by not releasing a v2 for it 😁